src/qz/utils/DPI.java

```diff
@@ -0,0 +1,44 @@
+//TODO find a better home and name for me
```

Comment:
I see two options for this - either we throw the class contents inside the `UbuntuUtilities` instead since that's the only location it is used, or we keep it here but call it `DisplayUtilities` and move relevant methods over from the `SystemUtilities` class too.
I like option 2.

Reply:
Thanks @bberenz.  I've done a bunch of refactoring.  Can you take a second look?